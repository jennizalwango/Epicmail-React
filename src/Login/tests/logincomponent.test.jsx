import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Login from '../components/LoginComponent';

describe('Login ', () => {
  let wrapper;
  const loginprops = {
    handleSubmit: jest.fn(),
    onChange: jest.fn(),
    email: '',
    isLoggingIn: false,
    password: '',
    errors: {},
  };
  it('renders the login page', () => {
    wrapper = shallow(<Login {...loginprops} />);
  });
  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
