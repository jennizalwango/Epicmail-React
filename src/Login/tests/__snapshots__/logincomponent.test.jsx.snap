// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login  matches snapshot 1`] = `
<div
  className="Login-container"
>
  <div
    className="heading"
  >
    <h1>
      Login
    </h1>
  </div>
  <div
    className="myform"
  >
    <form>
      <span />
      <div
        className="form-input"
        id="errormessage"
      >
        <input
          id="userEmail"
          name="email"
          onChange={[MockFunction]}
          placeholder="Email"
          required={true}
          type="text"
          value=""
        />
        <span />
      </div>
      <br />
      <div
        className="form-input"
        id="errormessage"
      >
        <input
          name="password"
          onChange={[MockFunction]}
          placeholder="Password"
          type="password"
          value=""
        />
      </div>
      <p
        className="mypara"
      >
        <a
          href="/reset"
        >
          Forgot Password?
        </a>
      </p>
      <br />
      <Component
        component="div"
        loaded={true}
        loadedClassName="loadedContent"
        parentClassName="loader"
      >
        <div
          className="form-input"
        >
          <button
            disabled={false}
            onClick={[MockFunction]}
            type="button"
          >
            <span>
              Login
            </span>
          </button>
        </div>
        <p
          className="mypara2"
        >
          Or signup with
        </p>
        <FacebookButton />
        <GoogleButton />
      </Component>
    </form>
  </div>
</div>
`;
